### PR TITLE
Update  GROMACS section README.md

### PR DESCRIPTION
**Pull request by cristianacarpinteiro**
Added some information to the GROMACS scenarios and visualization

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,59 +146,75 @@ coastal area.
...
-            nsteps_minim = 5000)
+```python
+output = scenario.simulate(simulation_time = 10, nsteps_minim = 5000)
 ```
 
-The simulate method initializes a simulation in the cloud. In this call, we set the parameters:
+The simulate method initializes a simulation using cloud resources. In this call, we set the following parameters:
  - simulation_time = 10: sets the trajectories time to span 10 **nanosecons**;
- -nsteps_minin = 5000: sets the number of minimization steps in the energy minimization step. 
+ - nsteps_minin = 5000: sets the number of minimization steps in the energy minimization step.
```
Comment (IvanPombo) [inline README.md:170]: I feel these defaults are weird here, but I will check over all.

File: README.md [hunk 2]
```diff
@@ -146,59 +146,75 @@ coastal area.
...
+
+The simulation output files can be found in inductiva_output/task_id folder.
```
Comment (IvanPombo) [inline README.md:172]: What does this path actually mean? If we are sync, then task_id is irrelevant.

**Review by IvanPombo — APPROVED** [review]
LgTm

File: README.md (hunk 2 above)
Comment (cristianacarpinteiro) [reply README.md:172]: I will remove this part.

File: README.md (hunk 1 above)
Comment (cristianacarpinteiro) [reply README.md:170]: I will take this part out too, in the next PR.

**Pull request merged**